### PR TITLE
do not modify unrelated api manager properties for test

### PR DESCRIPTION
**Pull request by pb82**
Only modify properties of the APIManager CR that are relevant for the test. Specifically, do not modify the value of the high availability flag because that would cause extra deployments in the 3Scale namespace.

### CODE REVIEW

File: test/common/scale_up_replicas_threescale.go [hunk 1]
```diff
@@ -76,48 +75,24 @@ func getAPIManager(dynClient k8sclient.Client) (threescalev1.APIManager, error)
...
-			Zync: &threescalev1.ZyncSpec{
-				AppSpec: &threescalev1.ZyncAppSpec{Replicas: &[]int64{replicas}[0]},
-				QueSpec: &threescalev1.ZyncQueSpec{Replicas: &[]int64{replicas}[0]},
-			},
-		},
-	}
+
+	apim.Name = name
+	apim.Namespace = namespace
+	apim.ResourceVersion = apim.GetResourceVersion()
```
Comment (matskiv) [inline test/common/scale_up_replicas_threescale.go:85]: Is the name, namespace and resourceversion added intentionally here?
I would expect those field to already be set in the CR we Get. 🤔

Comment (matskiv) [reply test/common/scale_up_replicas_threescale.go:85]: This is minor, and can be left as is.
I am approving, but it will merge after the release.

Comment (pb82) [reply test/common/scale_up_replicas_threescale.go:85]: I would also expect them to be present already. I can try to remove and rerun the tests.

**Comment by matskiv** [comment]
Cathal implemented patch in #754 
So seems like we can close this one.
Is that okay with you @pb82

**Comment by pb82** [comment]
@matskiv Yes, that's ok. Closing this one.

**Pull request closed without merging**